src/Multiselect.elm

```diff
@@ -214,7 +212,7 @@ update : Msg -> Model -> ( Model, Cmd Msg, Maybe OutMsg )
 update msg (Model model) =
     case msg of
         Start ->
-            (Model model, Cmd.none, Nothing )
+            ( Model model, Cmd.none, Nothing )
```

Comment:
seems like he didnt run elm-format lol

Reply:
yah... might want to turn off elm-format for these changes or do that separately or something. it makes the diff hard to read.